src/software/ai/hl/stp/play/free_kick_play.cpp

```diff
@@ -8,13 +8,20 @@
 #include "software/ai/hl/stp/tactic/move/move_tactic.h"
 #include "software/ai/hl/stp/tactic/receiver/receiver_tactic.h"
 #include "software/ai/passing/eighteen_zone_pitch_division.h"
+#include "software/ai/passing/sampling_pass_generator.h"
 #include "software/geom/algorithms/contains.h"
 #include "software/logger/logger.h"
 #include "software/util/generic_factory/generic_factory.h"
 #include "software/world/ball.h"
 
 FreeKickPlay::FreeKickPlay(TbotsProto::AiConfig config)
-    : Play(config, true), MAX_TIME_TO_COMMIT_TO_PASS(Duration::fromSeconds(3))
+    : Play(config, true),
+      MAX_TIME_TO_COMMIT_TO_PASS(Duration::fromSeconds(3)),
```

Comment:
why is this constant initialized like this?

Reply:
Not sure, it was there from before and this file will be overwritten in #2953